proposals/dark_mode_extension_icons.md

```diff
@@ -0,0 +1,231 @@
+# Dark Mode Extension Icons
+
+**Summary**
+
+Feature to enable developers to enhance extension icon visibility in dark mode.
+
+**Document Metadata**
+
+**Author:** solomonkinard
+
+**Sponsoring Browser:** Chromium, Safari, Firefox
```

Comment:
I think there was a thread about this, so apologies if I missed the resolution, but I think this should just be Chromium. This is not intended to represent the interested parties or even those that will implement, but whoever is specifically putting time into leading the proposal. No strong feelings on which vendor this is but I don't think having three makes sense. A good way of thinking about it is likely "which vendor should I contact for questions about the state of the proposal".

Reply:
Done.